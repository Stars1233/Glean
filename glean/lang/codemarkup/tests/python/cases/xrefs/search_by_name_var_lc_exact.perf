{
  "@generated": null,
  "facts_searched": { "python.SearchVariableByLowerCaseName.4": 1 },
  "full_scans": [ ]
}