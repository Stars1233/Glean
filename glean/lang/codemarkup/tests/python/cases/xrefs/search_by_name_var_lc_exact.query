query: python.SearchVariableByLowerCaseName { name_lowercase = "big_var" }
perf: true
transform: [gensort, []]
