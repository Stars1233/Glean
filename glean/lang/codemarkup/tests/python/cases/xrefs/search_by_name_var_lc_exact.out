[ "@generated", { "key": { "name_lowercase": "big_var", "name": "big_var" } } ]